### PR TITLE
Mesonupdate

### PR DESCRIPTION
**Pull request by brainstormtrooper**
Align meson build setup with standard GJS app specified by Gnome

### CODE REVIEW

File: data/meson.build [hunk 1]
```diff
@@ -15,11 +15,20 @@ desktop = intl.merge_file(
...
 meson.add_install_script('../meson/meson_post_install.py')
 
-appdatadir = join_paths(datadir, 'appdata')
+local_schemas = configure_file(copy: true,
+  input: gsettings_schema,
+  output: gsettings_schema)
+compile_local_schemas = custom_target('compile_local_schemas',
+  input: local_schemas,
+  output: 'gschemas.compiled',
+  command: ['glib-compile-schemas', meson.current_build_dir()])
```
Comment (eli-schwartz) [inline data/meson.build:29]: I think you probably want this? https://mesonbuild.com/Gnome-module.html#gnomecompile_schemas

File: meson.build [hunk 2]
```diff
@@ -1,51 +1,32 @@
...
 app_configuration.set('pkgdatadir', pkgdatadir)
 
-subdir('src')
-# subdir('src/lib')
-# subdir('src/ui')
-# subdir('src/object')
 subdir('data')
+subdir('src')
 subdir('po')
 meson.add_install_script('meson/meson_post_install.py')
```
Comment (eli-schwartz) [inline meson.build:32]: BTW you can drop this and use https://mesonbuild.com/Gnome-module.html#gnomepost_install

File: data/meson.build (hunk 1 above)
Comment (brainstormtrooper) [reply data/meson.build:29]: Indeed that does look like a better choice. Testing it today.

File: meson.build (hunk 2 above)
Comment (brainstormtrooper) [reply meson.build:32]: I have replaced the post-install script with this command. Will test today.
Thank you for your suggestions :-)

**Pull request merged**